client/INLClient.py

```diff
@@ -31,6 +33,10 @@ def __init__(self, client_info):
         self.modules = Modules.Modules()
         self.check_settings()
         self.client_info["servers"] = [ s[0] for s in settings.SERVERS ]
+        self.client_info["manage_build_root"] = settings.MANAGE_BUILD_ROOT
+        self.client_info["jobs_ran"] = 0
+        if self.get_client_info('manage_build_root'):
+            self.check_build_root()
```

Comment:
Do we really want this to only be an INL client feature?  Or is this general enough to just be a base client feature?

Reply:
I thought about that - but we actually only ever set BUILD_ROOT in INLClient, so I opted to do it all there instead